src/toolbars/MeterToolBar.cpp

```diff
@@ -183,6 +183,7 @@ void MeterToolBar::Populate()
        This is the name used in screen reader software, where having 'Meter' first
        apparently is helpful to partially sighted people.  */
       mRecordMeter->SetLabel( XO("Meter-Record") );
+      mRecordMeter->SetSliderName(XO("Recording Level"));
```

Comment:
Okay, at another commit, now I see the problems of previous commits corrected.  Names before the :, and a nice hover text with a rounded border.

In fact, now the old behavior of the pan and gain sliders looks worse!  There should be a minor issue I think to fix them to be consistent with the volume sliders.  Which is not part of the scope of this PR.

Reply:
It is still the case, however, that when double clicking the volume slider thumb to get a dialog, I'm still not entering a percentage value, but instead a fraction in the range 0.0 to 1.0.

Reply:
Thanks, dialog now shows values in range 0..100